easybuild/easyconfigs/a/ASE/ASE-3.16.0-iomkl-2018.02-Python-3.6.4.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'ASE'
+version = '3.16.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://wiki.fysik.dtu.dk/ase'
+description = """ASE is a python package providing an open source Atomic Simulation Environment
+ in the Python scripting language."""
+
+toolchain = {'name': 'iomkl', 'version': '2018.02'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
```

Comment:
@schiotz Please include a `SHA256` checksum (good way to re-trigger the tests once #6447 is merged)